### ISSUE TITLE
few-shot datas

### ISSUE THREAD

**Issue by Jacksong918** [MarcLafon/gallop#3]
Hello, I am very interested in your paper. Could you share the experimental data of the 11 datasets with few-shot samples? I would be very grateful.

**Comment by MarcLafon** [comment]
Hello, the data for the 11 datasets can be found here on the [CoOp](https://github.com/KaiyangZhou/CoOp/blob/main/DATASETS.md) repo. Then, we create the few-shot samples in the [train.py](https://github.com/MarcLafon/gallop/blob/main/gallop/train.py#L269C5-L269C102) file using this:
`train_dataset = dts_tools.create_few_shots_dataset(train_dataset, args.num_shots, seed=args.seed)`

I hope this answers your question.
